roles/update-access/test/authorized_keys

```diff
@@ -0,0 +1,2 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQD25J3HqZfjD2t9oCCZwrTCQRjR07q9FJlw49IT8IQPVd5ODTx7nlft0vnBYrGMBp55J6pDzdd9W5SllxpjGhF4b+bARwS/6Qg6kZEG9PrpvgwxfcXQ0BqAb871kJfQVac9hSm7qJ5I++Gx2ytuIVXnSn8UlIsbhrOgcuvrlUY/5ahZ7n4w03CFD2kgQyBvm1WUuNd2jX6CET+0Ke/hYKqTYSad15bCMybIabdCtzGpib9VX049BPkW68ZjR+f9XcCTCjEwuq20JzD24OpdtdRdu/JyYIGTLntaD9buz1Sktni4tFKkEioJ9GrOmS2EaWSKC9lw9yEkNsSsHy/zRvtl user1@example.com
```

Comment:
please include a couple comments with the corresponding private key(s) so we know how this can be tested.